components/PlantCard/index.js

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image"
```

Comment:
I noticed that you imported next/image, but it is not being utilized in your code